compiler/test/compilable/issue16020.d

```diff
@@ -19,21 +20,13 @@ alias void F51() @system;
 alias F52 = void() @safe;
 static assert (!is(F51 == F52));
 
-alias F61 = int() const shared;
-alias int F62() const shared ;
-alias F63 = const shared int();
-static assert (is(F61 == F62));
-static assert (is(F63 == F62));
-
-alias F71 = int() immutable inout;
-alias int F72() immutable inout;
-alias F73 = immutable inout int();
-static assert (is(F71 == F72));
-static assert (is(F73 == F72));
-
```

Comment:
stupid question: why are these removed?

Reply:
I've reverted this now. I forgot that a method type can have TypeCtor attributes.